### DISCUSSION TITLE
Transaction export

### DISCUSSION THREAD

**Discussion by cknepper1** [alephium/desktop-wallet#180] [Ideas] [answered]
Does anyone know how to export the transaction data from the Alephium wallet? ...or when that feature will be added?  Thank you.

**Comment by nop33** [comment, marked as answer]
Currently there's no such feature. Feel free to create a [feature request](https://github.com/alephium/desktop-wallet/issues/new/choose)!

**Comment by Ploddles** [comment]
I asked about this when the first wallet was released and was told that there was no intention, at least at that time, to add this feature.

**Reply by lf94** [reply]
Hm, not sure why there'd be no intention originally but I'd like to see this too!

**Reply by mvaivre** [reply]
We will start by adding the feature in the explorer first, as described [here](https://github.com/alephium/explorer/issues/41). Then it should be straightforward to implement it in the desktop wallet.

**Reply by ShkarShow** [reply]
Can i create a wallet from here?

**Comment by ShkarShow** [comment]
Hellow guys
How i can create a wallet from here?

**Reply by mvaivre** [reply]
@ShkarShow Hi! Just download the latest release here: https://github.com/alephium/desktop-wallet/releases/tag/v1.5.2
